src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -81,26 +81,59 @@ describe('InfiniteScroll', () => {
   });
 
   test('mixed items', () => {
```

Comment:
Trying to move tests to explicitly state what the purpose of the test is by supplying in the format:
"should [expectation] when [scenario]".

[Related discussion here](https://github.com/grommet/grommet/pull/4390#discussion_r468112613)

```suggestion
  test('should render expected items when supplied assortment of mixed items', () => {
```